credentials/bc-petroleum-and-natural-gas-title/vocabulary.md

```diff
@@ -0,0 +1,98 @@
+# Vocabulary
+
```

Comment:
We should include a reference to the source of truth of these vocab entries, in this case, I believe it is the Act.